OpenProblemLibrary/ASU-topics/setCalculus/stef/stef2_5p4.pg

```diff
@@ -28,34 +29,32 @@ DOCUMENT();        # This should be the first executable line in the problem.
 
 loadMacros(
   "PGstandard.pl",
-  "PGchoicemacros.pl",
-  "extraAnswerEvaluators.pl",
-  "MathObjects.pl",
+  "PGML.pl",
   "PGcourse.pl"
 );
 
-$b = random(1,8,1);
+Context("Interval");
+$b = random(1,8);
+$answer= Interval("[$b,Inf)");
+$f= Formula("sqrt(x-$b)");
 
-TEXT(beginproblem());
 
-$showPartialCorrectAnswers = 1;
+BEGIN_PGML
 
-TEXT(EV3(<<'EOT'));
-Let
-\[ f(x) = \sqrt{x - $b}.  \]
-Use \{helpLink("interval notation")\} to indicate where \( f \) is continuous.
+Let [`f(x)`] =  [`[$f]`].   Use [@ helpLink("interval notation") @]*  to indicate where [`f(x)`] is continuous.
 
-$BR$BR
+Domain of continuity:[___________]{$answer}{15}
 
-Domain of continuity:
-\{ans_rule(35)\}
-EOT
+END_PGML
 
-# @answers = "[$b,Inf)";
-# ANS(interval_cmp(@answers) );
+$showHint=3;
+BEGIN_PGML_HINT
+The domain of a square root function is the set of all real numbers for which the function is defined. Therefore, the expression [`x`]-[$b] inside the square root [` \sqrt{} `] must not be negative.
+END_PGML_HINT
 
-Context("Interval");
-$ans = Compute("[$b,Inf)");
-ANS($ans->cmp);
 
-ENDDOCUMENT();
+BEGIN_PGML_SOLUTION
+The square root function is continuous on its domain. So [`x`]-[$b] >= 0. Thus [`x >=`] [$b] which is [`[$answer]`] in interval notation.
```

Comment:
Put all math in math mode.  You can replace
```
[`x`]-[$b] >= 0 
```
with 
```
[` x - [$b] \geq 0 `]
```

The same with the one after "Thus"